log/log.go

```diff
@@ -0,0 +1,47 @@
+package log
+
+import (
+	"io/ioutil"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var log *logrus.Logger
+
+const DebugLevel = logrus.DebugLevel
+
+func Debugf(format string, args ...interface{}) {
+	log.Debugf(format, args...)
```

Comment:
>  Also I would like to avoid using global variables here.

Not sure how we could do that, apart from reintroducing something like the context logger that was removed in #50.